File: downloads/code/LeetCode/Java/Length-of-Last-Word.java
public class Solution {
    public int lengthOfLastWord(String s) {
        if (s.length()==0)
            return 0;
        // 去掉首尾空格
        s = s.trim();
        int lastIndex = s.lastIndexOf(' ')+1;
        return s.length()-lastIndex;
    }
}

File: downloads/code/LeetCode/Python/Length-of-Last-Word.py
class Solution(object):
    def lengthOfLastWord(self, s):
        """
        :type s: str
        :rtype: int
        """

        if not s:
            return 0
        strs = s.split()
        # " "
        if len(strs) == 0:
            return 0
        return len(strs[-1])
